lib/OpenQA/WebAPI/Plugin/AMQP.pm

```diff
@@ -104,6 +105,12 @@ sub on_job_event ($self, $args) {
         if ($event_data->{bugref} = $bugref) {
             $event_data->{bugurl} = OpenQA::Utils::bugurl($bugref);
         }
+        # include failed test modules
+        my $tmo = read_test_modules($job);
+        my $test_modules = $tmo ? $tmo->{modules} : [];
+        my @faileds = grep { $_->{result} eq OpenQA::Jobs::Constants::FAILED } @$test_modules;
+        my @fnames = map { $_->{name} } @faileds;
+        $event_data->{failedmodules} = @faileds ? \@fnames : [];
```

Comment:
```suggestion
        $event_data->{failed_modules} = $job->failed_modules;
```

Reply:
thanks. I was away yesterday so didn't get to look at it, will verify and update the PR tomorrow.

Reply:
> And underscores are a pain to type. :D

For that I recommend https://neo-layout.org/, works well for German and English and Perl and Python :)

Reply:
thanks, done.